rotkehlchen/exchanges/coinbase.py

```diff
@@ -160,20 +172,106 @@ def __init__(
             secret=secret,
             database=database,
         )
+
+        self.is_legacy_api_key: bool | None = None
```

Comment:
Hmm @vijithreddy okay then this only runs when the exchange is edited or setup. Then add it here too **but make sure to add it under try catch for the errors the function adds**. And log an error if it happens and set it to legacy.

as I wrote here: https://github.com/rotki/rotki/pull/7903#discussion_r1596391668

Reply:
Added the logic and logging error. 

Also believe I fixed frontend lint issues. And tests failed because of a missing PyJWT module, added that to the requirements.